dev_cmds/build-all.ts

```diff
@@ -1,7 +1,6 @@
-import { execPiped } from "devcmd";
+import { execPiped, runAsyncMain } from "devcmd";
```

Comment:
since the build scripts (i.e. `dev_cmds`) in this repo use the released version of DevCmd, not the development version, we can't directly use the new functions here without publishing a new release.

this means for now, we need to:
* revert the changes to `devcmd/`
* publish v0.0.7
* then update `devcmd/package.json` to refer to the new version, and use the new function